contrib/templates/experimental/scala-job/databricks_template_schema.json

```diff
@@ -0,0 +1,40 @@
+{
+  "properties": {
+    "project_name": {
+      "type": "string",
+      "description": "Name of project. Do not add spaces",
+      "order": 1
+    },
+    "custom_workspace_host": {
+      "type": "string",
+      "default": "{{workspace_host}}",
```

Comment:
Could you follow the convention of https://github.com/databricks/cli/blob/main/libs/template/templates/default-sql/databricks_template_schema.json#L2 here, where the URL is just printed and not editable? When it's editable, we can't really derive other properties from the current session anymore, like the current user name, catalog support, or cloud